server/methods/methods_api.py

```diff
@@ -0,0 +1,118 @@
+import asyncio
+from dataclasses import dataclass, field
+from concurrent.futures import ProcessPoolExecutor
+
+import aiohttp
+import itk
+
+itk.force_load()
+
+from volview_server import (
+    VolViewApi,
+    get_current_client_store,
+    get_current_session,
+)
+
+from volview_server.transformers import (
+    convert_itk_to_vtkjs_image,
+    convert_vtkjs_to_itk_image,
+)
+
+from methods_utils import (
+    show_image,
+    CommonClientState,
+)
+
+from median_filter_method import (
+    MedianFilterClientState,
+    median_filter_method,
+)
+
+## Link to app ##
+
+volview = VolViewApi()
+
+## server-based methods ##
+
+process_pool = ProcessPoolExecutor(4)
+
+## Compose the Client State ##
+
+
+class CurrentClientState:
+    def __init__(self):
+        self.common = CommonClientState()
+        self.median_filter = MedianFilterClientState()
+
+
+## The median filter is allocated to a process ##
+async def run_median_filter_process(process_pool, state):
```

Comment:
The benefit of having this in a separate function isn't clear to me.  Seems like these few lines could be moved into the caller.  Admittedly the caller calls this function using an await, but I don't think that nesting of awaits are needed.

Reply:
There's no reason to have this in a separate function. I wrote the usage before the implementation, and I never did a clean-up pass.